### PR TITLE
LPS-157587 | Add supported query params to Headless Delivery API Explorer

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Add a test to assert to support query params to Headless Delivery API Explorer

**Test Plan**
**Given** a blog-posting created
**When** with curl I request GET getSiteBlogPostingsPage with siteId and restrictFields = "aggregateRating,alternativeHeadline,articleBody,creator,customFields,dateCreated,dateModified,datePublished,
description,encodingFormat,externalReferenceCode,friendlyUrlPath,headline,image,keywords,
numberOfComments,relatedContents,renderedContents,siteId,taxonomyCategoryBriefs,x-class-name,facets,actions"
**Then** in a response I receive a with id field values only

**Given** an Asset Library with a Web Content created
**When** with curl I request GET getAssetLibraryContentElementsPage with assetLibraryId and fields = "content.assetLibraryKey"
**Then** in a response I receive a body with assetLibraryKey only

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-157587

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-frontend-functional:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/BlogPostingAPI.macro [hunk 1]
```diff
@@ -292,6 +300,23 @@ definition {
 		return "${curl}";
 	}
 
+	macro getBlogPostingsbyDifferentParameters {
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/BlogPostingAPI.macro:303]: very generic macro, nice! 👏🏻

**Review by magjed4289 — APPROVED** [review]
Looks good! :)

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-frontend-functional:nocompile

**Review by yanan — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**